arangod/Agency/Node.cpp

```diff
@@ -1148,3 +1138,14 @@ void Node::clear() {
     return _parent->removeChild(_nodeName);
   }
 }
+
+auto Node::getUIntAsIntWithDefault(Slice slice, std::string_view key, std::int64_t def)
+    -> std::int64_t {
+  if (slice.isObject()) {
+    Slice value = slice.get(key);
+    if (value.isUInt()) {
```

Comment:
```suggestion
    if (value.isInt()) {
```

Reply:
I have the feeling that this is intended, in the sense that the step may not be a negative number. I'll check that with Kaveh.

Reply:
It's not, I fixed it.